alerting/notifier/channels/kafka.go

```diff
@@ -48,12 +53,25 @@ type KafkaNotifier struct {
 }
 
 type kafkaSettings struct {
-	Endpoint    string `json:"kafkaRestProxy,omitempty" yaml:"kafkaRestProxy,omitempty"`
-	Topic       string `json:"kafkaTopic,omitempty" yaml:"kafkaTopic,omitempty"`
-	Description string `json:"description,omitempty" yaml:"description,omitempty"`
-	Details     string `json:"details,omitempty" yaml:"details,omitempty"`
+	Endpoint       string `json:"kafkaRestProxy,omitempty" yaml:"kafkaRestProxy,omitempty"`
+	Topic          string `json:"kafkaTopic,omitempty" yaml:"kafkaTopic,omitempty"`
+	Description    string `json:"description,omitempty" yaml:"description,omitempty"`
+	Details        string `json:"details,omitempty" yaml:"details,omitempty"`
+	BasicAuthUser  string `json:"basicAuthUser,omitempty" yaml:"user,omitempty"`
```

Comment:
can it be Username and JSON\yaml `username`?

Reply:
On it!

Reply:
Done!